File: Data Structures & Algorithms/two-integer-sum/submission-2.go
func twoSum(nums []int, target int) []int {
    seen := make(map[int]int)
    for idx,val := range nums{
       if res,ok := seen[target-val];ok{
        return []int{res,idx}
       }
       seen[val] = idx
    }
    return nil
}

File: Data Structures & Algorithms/concatenation-of-array/submission-2.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        numLen = len(nums)
        res = [0] * numLen*2
        for i in range(len(nums)):
            res[i] = res[i+numLen] = nums[i]
        return res